src/Game/Maps.elm

```diff
@@ -187,6 +189,18 @@ currentAreaMap (A model) =
                 Dict.empty
 
 
+{-| Get the width and height of the map for the current area
+-}
+currentAreaSize: Maps -> Size
```

Comment:
It is better to do something like `mapSize : Map -> Size` as then it's not tied in to the current area and you can get the size of any map. Then you would pass in the map using `currentAreaMap`.